.github/workflows/build.yml

```diff
@@ -83,6 +83,14 @@ jobs:
         run: python .github/scripts/checkstyle.py
         env:
           GITHUB_WORKSPACE: ${{ github.workspace }}
+      - name: Update Floating 'cached' Tag on Successful Trunk Build
+        if: success()
```

Comment:
This workflow is used by PRs as well as trunk. We need to make sure we're only tagging the commits on trunk. You could do that here with an additional condition, or add a trunk-only step to ci.yml